src/components/CompanyMatchPage/MatchCard.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react'
+import { Image, Badge, Button,Form, FormControl} from "react-bootstrap";
+import male from "../../assets/images/male.png";
+
+
+const MatchCard = () => {  
+  
+  return(
+    
+    
+        <div className="Container">
+          <div className="d-flex flex-row-reverse">
+  <Form inline className="search-form">
+    <FormControl type="text" placeholder="Search" className=" mr-sm-2" />
+    
+  </Form>
```

Comment:
The search form can't be inside of a card, because otherwise it will show up above every card on that page. 

The CompanyMatchPage should be at the pages folder and MatchCard at the components folder.

Reply:
fixed